pennylane_lightning/src/tests/Test_RuntimeInfo.cpp

```diff
@@ -0,0 +1,13 @@
+#include "Macros.hpp"
+#include "RuntimeInfo.hpp"
+
+#include <catch2/catch.hpp>
+
+using namespace Pennylane::Util;
+
+TEST_CASE("Runtime information is correct", "[Test_RuntimeInfo]") {
+    INFO("RuntimeInfo::AVX " << RuntimeInfo::AVX());
+    INFO("RuntimeInfo::AVX2 " << RuntimeInfo::AVX2());
+    INFO("RuntimeInfo::AVX512F " << RuntimeInfo::AVX512F());
+    REQUIRE(true);
+}
```

Comment:
This is used to report the runtime info in the test suite?

Reply:
I added this just to see whether it works correctly (just ran on different devices and watched it is correct) but can be used to report runtime info if needed.